wallets/provider-ledger/src/helpers.ts

```diff
@@ -0,0 +1,54 @@
+import AppEth from '@ledgerhq/hw-app-eth';
+import TransportU2F from '@ledgerhq/hw-transport-webhid';
+
+const ETHEREUM_CHAIN_ID = '0x1';
+
+export const ETH_BIP32_PATH = "44'/60'/0'/0/0";
+
+const ledgerErrorMessages: { [statusCode: number | string]: string } = {
+  21781: 'The device is locked',
+  25871: 'Related application is not ready on your device',
+  27013: 'Action denied by user',
+  INSUFFICIENT_FUNDS: 'Insufficient funds for transaction',
+};
+
+export function getLedgerInstance() {
+  return { chainId: ETHEREUM_CHAIN_ID };
```

Comment:
Please add a comment to describe why it's like this here. (We've talked offline, writing it down for documenting purposes)